packages/antd/src/hooks/form/useDrawerForm/useDrawerForm.ts

```diff
@@ -102,15 +109,15 @@ export const useDrawerForm = <
             ? `drawer-${resource?.identifier ?? resource?.name}-${action}`
             : undefined;
 
-    const [open, setOpen] = useState(false);
-
     React.useEffect(() => {
         if (initiallySynced.current === false && syncWithLocationKey) {
             const openStatus = parsed?.params?.[syncWithLocationKey]?.open;
```

Comment:
IF I'm not missing something, can't we refactor this like following suggestion?

```suggestion
            const openStatus = Boolean(parsed?.params?.[syncWithLocationKey]?.open);
```

Reply:
We can, but the code is handled for both boolean and string values. So it's okay if we don't.